tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -440,19 +439,14 @@ public void testSimpleWebAppWithWebsocket(String arg) throws Exception
             try (DistributionTester.Run run2 = distribution.start(args2))
             {
                 assertTrue(run2.awaitConsoleLogsFor("Started Server@", 10, TimeUnit.SECONDS));
-                // we do not test that anymore because it doesn't work for java14
-                //assertFalse(run2.getLogs().stream().anyMatch(s -> s.contains("LinkageError")));
+                assertFalse(run2.getLogs().stream().anyMatch(s -> s.contains("LinkageError")));
 
                 startHttpClient();
                 ContentResponse response = client.GET("http://localhost:" + port + "/test1/index.jsp");
-                assertEquals(HttpStatus.OK_200, response.getStatus());
-                assertThat(response.getContentAsString(), containsString("Hello"));
-                assertThat(response.getContentAsString(), not(containsString("<%")));
+                assertEquals(HttpStatus.SERVICE_UNAVAILABLE_503, response.getStatus());
 
                 client.GET("http://localhost:" + port + "/test2/index.jsp");
-                assertEquals(HttpStatus.OK_200, response.getStatus());
-                assertThat(response.getContentAsString(), containsString("Hello"));
-                assertThat(response.getContentAsString(), not(containsString("<%")));
+                assertEquals(HttpStatus.SERVICE_UNAVAILABLE_503, response.getStatus());
```

Comment:
I had the idea about adding another webapp to be sure it's still deployed despite having this LinkageError

Reply:
The MethodHandle changes in this PR fix the LinkageError, but now this particular webapp should fail at deployment time anyway as the signature is wrong.

Reply:
Should I add in another webapp here to test that it deploys successfully even if these two fail?

Reply:
yes. I would even say 2 :). I'm not sure about deployment order of contexts. maybe one first and one last. to ensure the failure do not break anything else to be deployed.

Reply:
@lachlan-roberts great tests changes! we now even have real websocket tests in the distribution tests module!